olp-cpp-sdk-dataservice-write/include/olp/dataservice/write/StreamLayerClient.h

```diff
@@ -79,6 +79,21 @@ class DATASERVICE_WRITE_API StreamLayerClient {
                     StreamLayerClientSettings client_settings,
                     client::OlpClientSettings settings);
 
+  /**
+   * @brief Cancels all pending operations started by this client.
+   * Cancels all on-going publish operations, which are started by this client.
+   * This operation returns instantly without waiting for the callbacks on these
+   * operations.
+   *
+   * This operation can be useful if you want to cancel all the submitted
+   * publish requests without destroying actual client instance.
+   * @note This operation won't cancel queued publish requests via \ref Queue
+   * method - in order to do that, you have to clear the \ref
```

Comment:
Please remove the "- in order to do that .... " part as we will implement a method to do that also.
With the current cache key setup user will have a quite hard time to delete the queued publishes.